release_task.rb

```diff
@@ -143,13 +149,13 @@ def define_version_update_task
     namespace :version do
       desc "Update version"
       task :update do
-        latest_release = GitHubClient.new(@product, @product).latest_release
-        # "Groonga 14.1.1 - 2024-12-03" or "Mroonga 14.11 - 2024-12-03"
-        release_name = latest_release["name"]
-        # "14.1.1" or "14.11"
-        latest_version = release_name[/\d+(\.\d+){1,2}/, 0]
+        latest_tag = GitHubClient.new(@product, @product).latest_tag
+        # "v14.1.3"(Groonga), "v14.14"(Mroonga) or "3.2.5"(PGroonga)
+        release_tag_name = latest_tag["name"]
+        # "14.1.3", "14.14" or "3.2.5"
+        latest_version = release_tag_name[/\d+(\.\d+){1,2}/, 0]
         # "2024-12-03"
-        latest_release_date = release_name[/\d+-\d+-\d+/, 0]
+        latest_release_date = Date.today.to_s
```

Comment:
We couldn't get the release date from the response of `/tag` endpoint.
Therefore, I decided to use the current date since it’s likely to match the actual release date.

Reply:
It looked like we could get it from the commit.
https://api.github.com/repos/groonga/groonga/commits/76baa607025cfbd54d9f1e60a11939a9e186d760

Reply:
Can we use https://docs.github.com/en/rest/git/commits?apiVersion=2022-11-28#get-a-commit-object ? It seems that https://api.github.com/repos/groonga/groonga/tags include URLs for the API endpoint.

Reply:
fix: [bdf77ea](https://github.com/groonga/groonga.org/pull/95/commits/bdf77ead761bc5bfc51dc0435b056d66d16735d4) Thanks you so much. I took a release date from the commit.